cfgov/unprocessed/apps/rural-or-underserved-tool/js/get-rural-counties.js

```diff
@@ -1,15 +1,26 @@
 require( 'es6-promise' ).polyfill();
 import { fetch } from 'cross-fetch';
 
+
+const cache = {};
+
 /**
  * @param {string} year - A year.
- * @returns {Object} Result of the call to a JSON file.
+ * @returns {array} An array of rural counties by fips.
  */
 function getRuralCounties( year ) {
+  if ( cache[year] ) return Promise.resolve( cache[year] );
```

Comment:
What testing condition triggers this?

Reply:
Well spotted! This actually currently isn't triggered by any code due to the weird routing operations of the tool. Really, instead of the "start a new search" button reloading the page via navigation, it should just wipe the state/reshow the search components. I made this change so that if we ever properly redesign this tool, it will have this simple caching logic.

Reply:
Might be worth a comment above it saying this.